tests/src/CLRTest.Execute.Batch.targets

```diff
@@ -46,6 +46,35 @@ if not exist "$(MSBuildProjectName).ni.exe" "%CORE_ROOT%\crossgen.exe" /Platform
     </PropertyGroup>
   </Target>
```

Comment:
This is for Windows only and will follow UNIX change in the next check in after testing.

Sent from Outlook<http://aka.ms/Ox5hz3>




On Wed, Dec 30, 2015 at 6:03 PM -0800, "Jan Kotas" <notifications@github.com<mailto:notifications@github.com>> wrote:


In tests/src/CLRTest.Execute.Batch.targets<https://github.com/dotnet/coreclr/pull/2487#discussion_r48643999>:

> @@ -46,6 +46,35 @@ if not exist "$(MSBuildProjectName).ni.exe" "%CORE_ROOT%\crossgen.exe" /Platform
>      </PropertyGroup>
>    </Target>


Is a matching change needed in CLRText.Execute.Bash.targets for Unix?

-
Reply to this email directly or view it on GitHub<https://github.com/dotnet/coreclr/pull/2487/files#r48643999>.